console/Makefile

```diff
@@ -13,12 +13,6 @@ ifneq ($(filter aarch%,$(UNAME_P)),)
 	TARGET_ARCH = arm64
 endif
 
-ifdef CIRCLE_TAG 
```

Comment:
Have we confirmed that removing this is still sufficient for both tagged releases and simple PR SHA tags?

Reply:
Yep. What we are doing now is building the image with the sha and once all the release steps pass and we get sign off, the `console-docker-push-tag` job gets triggered. This takes the images that were created and pushed with the SHA and creates a new manifest using the provided circle tag.